### PR TITLE
Add JSON validation using jq in Workflow

### PR DESCRIPTION
**Pull request by zeegtab**
Added JSON validation using jq in the test job of our workflow.

I added jq as a dependency and created a validation script so that jq validates every json file under /tests/books.

Fixes: #25.

### CODE REVIEW

File: scripts/validate_json.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+for FILE in `ls ./tests/books`
+do
+  jq . "tests/books/$FILE"
+  exit_code=$?
+    if [ $exit_code -ne 0 ]; then
+      echo "$FILE could not be parsed."
```
Comment (JamesLee-Jones) [inline scripts/validate_json.sh:8]: You will probably then want to exit with the exit code that jq returned using `exit $exit_code`

File: scripts/validate_json.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (JamesLee-Jones) [inline scripts/validate_json.sh:1]: This file should be renamed to match the naming convention of the python check scripts, so `check_json_formatting.sh`.

**Review by JamesLee-Jones — APPROVED** [review]
Looks good, after addressing the comment, feel free to merge!

File: scripts/validate_json.sh (hunk 1 above)
Comment (zeegtab) [reply scripts/validate_json.sh]: Done.

File: scripts/validate_json.sh (hunk 2 above)
Comment (zeegtab) [reply scripts/validate_json.sh]: Done.

**Pull request merged**